### DISCUSSION TITLE
ESG for personal finance simulation?

### DISCUSSION THREAD

**Discussion by ktegan** [JuliaActuary/EconomicScenarioGenerators.jl#42] [General]
Hello!  As a personal project I wanted to write a proof of concept personal finance simulator that would compute taxes and show graphs of various investment strategies.  I have simple code to compute basic taxes for federal + state/local taxes in New York City, and I have ideas for a few simple investment strategies to test and graph.  Is EconomicScenarioGenerators.jl ready to generate approximate monthly values for the the below categories?  If so any suggestions on "reasonable" models and parameters to get started would be very much appreciated.  Really the only required values would be inflation, equity returns and short-term interest rate (ideally with some modeling of co-movement between inflation and interest rates).

 - inflation (ideally split between shelter and non-shelter)
 - equity returns (ideally with dividend yield)
 - t-bond and corporate bond investment cashflows
 - mortgage rates
 - real-estate returns (ideally both single family and investment)

Thanks!

**Comment by alecloudenback** [comment]
Hi there! I think the answer is yes and no... ESGs are a complicated area and it depends so much on your assumptions. This package can model equity returns and short term interest rates (including correlating scenarios with copulas) but you would have to specify your own assumptions.

**Reply by ktegan** [reply]
Thanks for the reply.  For now I just want to get this thing off the ground, so whatever assumptions lead to the simplest implementation (that could be improved later) and still produce somewhat plausible results would be my goal.  For equities returns / short term interest rates / inflation rates do you have any suggestions of models possibly with general parameters or ranges?  Thanks!

**Reply by ktegan** [reply]
Also if there are any outside tools (AAA ESG excel file?) that is already calibrated to give reasonable results and would let me create 1M scenarios for testing purposes that would also be very valuable.
